eng/common/scripts/Package-Properties.ps1

```diff
@@ -161,7 +188,7 @@ function Get-PkgListFromYml ($ciYmlPath)
 {
     $ProgressPreference = "SilentlyContinue"
     Register-PSRepository -Default -ErrorAction:SilentlyContinue
-    Install-Module -Name powershell-yaml -RequiredVersion 0.4.1 -Force -Scope CurrentUser
+    # Install-Module -Name powershell-yaml -RequiredVersion 0.4.1 -Force -Scope CurrentUser
```

Comment:
This will be needed If the module isn't installed. I think some more work need to be done here to make sure we work in both cases where it is installed and cases where it isn't.

Reply:
It super slow for having this in my local build. Since this is daily work, I don't want to distract my work of improving this. Comment out just for testing purpose. Will add back once ready to check in.

Reply:
Ok it would be worth seeing what better options we have we don't want it to be slow. I suspect if we remove the `-Force` option it should be a no-op after the module is already installed.